tests/test-commands.c

```diff
@@ -163,16 +168,38 @@ int main(void)
           Prepare to run the path management generic netlink command
           tests.
         */
-        struct l_idle *const idle =
-                l_idle_create(idle_callback, &pm, NULL);
+        bool tests_called = false;
+        struct l_genl *const genl = l_genl_new();
+        assert(genl != NULL);
+
+        assert(l_genl_add_family_watch(genl,
+                                       MPTCP_GENL_NAME,
+                                       run_tests,
+                                       NULL,
+                                       &tests_called,
+                                       NULL) != 0);
+
+        // Bound the time we wait for the tests to run.
+        static unsigned long const milliseconds = 500;
+        struct l_timeout *const timeout =
+                l_timeout_create_ms(milliseconds,
+                                    timeout_callback,
+                                    &tests_called,
+                                    NULL);
 
         (void) l_main_run();
 
+        /*
+          The tests will have run only if the "mptcp" generic netlink
+          family appeared.
+         */
+        assert(tests_called);
+
+        l_free(timeout);
```

Comment:
Better to call `l_timeout_remove()` here - will clean up more thoroughly and free `timeout`

Reply:
Done.